### DISCUSSION TITLE
HTML as an output file

### DISCUSSION THREAD

**Discussion by MatthewSkingley** [gotenberg/gotenberg#663] [Q&A] [answered]
Hi, is it possible to convert input file types to HTML files (or any other file types that are not PDF) using the options already provided by unoconv?

**Comment by gulien** [comment, marked as answer]
Hello @MatthewSkingley,

It is not currently supported, and I don't think I'll work on that kind of support (if even possible). Gotenberg is mostly about PDF files, and I want to keep it that way 😉
